packages/go/util/analyze.go

```diff
@@ -104,84 +103,6 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	se := string(rf)
-
-	var files []string
-	var relatedFiles []string
-
-	// Add entrypoint to watchlist
-	relFileName, err := filepath.Rel(filepath.Dir(fileName), fileName)
-	if err != nil {
-		log.Fatal(err)
-	}
-	relatedFiles = append(relatedFiles, relFileName)
-
-	// looking for all go files that have export func
-	// using in entrypoint
-	err = filepath.Walk(filepath.Dir(fileName), visit(&files))
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	// looking related packages
-	var modPath string
-	flag.StringVar(&modPath, "modpath", "", "module path")
-	flag.Parse()
-	if len(modPath) > 1 {
-		err = filepath.Walk(modPath, visit(&files))
-		if err != nil {
-			log.Fatal(err)
-		}
-	}
-
-	for _, file := range files {
-		absFileName, _ := filepath.Abs(fileName)
-		absFile, _ := filepath.Abs(file)
-		// if it isn't entrypoint
-		if absFileName != absFile {
-			// find all export structs and functions
-			pf := parse(file)
-			var exportedDecl []string
-
-			ast.Inspect(pf, func(n ast.Node) bool {
-				switch t := n.(type) {
-				case *ast.FuncDecl:
-					if t.Name.IsExported() {
-						exportedDecl = append(exportedDecl, t.Name.Name)
-					}
-				// find variable declarations
-				case *ast.TypeSpec:
-					// which are public
-					if t.Name.IsExported() {
-						switch t.Type.(type) {
-						// and are interfaces
-						case *ast.StructType:
-							exportedDecl = append(exportedDecl, t.Name.Name)
-						}
-					}
-				}
-				return true
-			})
-
-			for _, ed := range exportedDecl {
-				if strings.Contains(se, ed) {
-					// find relative path of related file
-					var basePath string
-					if modPath == "" {
-						basePath = filepath.Dir(fileName)
-					} else {
-						basePath = modPath
-					}
-
-					rel, err := filepath.Rel(basePath, file)
```

Comment:
This `basePath` is a relative path, but it should be absolute. Rather than fix this and wait to discover other edge cases in this unused logic, it has been removed.